frontend/lib/norent/letter-builder/confirmation.tsx

```diff
@@ -1,26 +1,147 @@
 import React, { useContext } from "react";
 import Page from "../../ui/page";
 import { AppContext } from "../../app-context";
+import { OutboundLink } from "../../analytics/google-analytics";
+import {
+  getUSStateChoiceLabels,
+  USStateChoice,
+} from "../../../../common-data/us-state-choices";
+import { LetterBuilderAccordion } from "./welcome";
+
+const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
+
+const NORENT_FEEDBACK_FORM_URL = "https://airtable.com/shrrnQD3kXUQv1xm3";
+const CANCEL_RENT_PETITION_URL = "https://cancelrent.us/";
 
 export const NorentConfirmation: React.FC<{}> = () => {
   const { session } = useContext(AppContext);
   const letter = session.norentLatestLetter;
+  const user = session.norentScaffolding;
+  const stateName =
+    user?.state && getUSStateChoiceLabels()[user?.state as USStateChoice];
 
   return (
-    <Page
-      title="Your letter has been sent!"
-      withHeading="big"
-      className="content"
-    >
-      <p>Noice.</p>
-      {letter?.trackingNumber ? (
-        <p>
-          Your letter was sent on {letter?.letterSentAt}. Its tracking number is{" "}
-          {letter?.trackingNumber}.
+    <Page title="You've sent your letter" className="content">
+      <div className="media">
+        <div className="media-left">
+          <i className="has-text-info">{checkCircleSvg}</i>
+        </div>
+        <div className="media-content">
+          <h2 className="title">You've sent your letter</h2>
+        </div>
+      </div>
+      <p>
+        Your letter has been sent to your landlord via email. A copy of your
+        letter has also been sent to your email.
+      </p>
+      {letter?.trackingNumber && (
+        <p className="is-size-4 has-text-weight-bold">
+          Tracking #: {letter?.trackingNumber}.
```

Comment:
This isn't super high priority, but (A) we might want to specifically mention that it's a USPS tracking number (as opposed to fedex, ups, etc) and (B) we might want to link to the actual USPS site.  we currently do this in `loc-confirmation.tsx` for the letter of complaint, we should probably just factor it out into a common location and reuse it here!

also the tracking number is super long.  we might want to make sure it can actually be displayed on tiny screens with out doing something weird, like making the page wider than the screen and creating an annoying horizontal scrollbar situation.

Reply:
Those are both great points! Let's bring that up with the team in QA